AndroidAnnotations/androidannotations/src/main/java/org/androidannotations/holder/EFragmentHolder.java

```diff
@@ -134,7 +136,7 @@ private void setFindViewById() {
 	}
 
 	private void setFragmentBuilder() throws JClassAlreadyExistsException {
-		fragmentBuilderClass = generatedClass._class(PUBLIC | STATIC, "FragmentBuilder_");
+		fragmentBuilderClass = generatedClass._class(PUBLIC | STATIC, "FragmentBuilder" + classSuffix());
```

Comment:
why? its an generated class that is accessible for the user. so `FragmentBuilderImpl` is better than `FragmentBuilder_` actually i'd rename the [`FragmentBuilder`](https://github.com/excilys/androidannotations/blob/326fe9666df37284245c4e2706f4c042f0b396c1/AndroidAnnotations/androidannotations-api/src/main/java/org/androidannotations/api/builder/FragmentBuilder.java) api class (to e.g. `AbstractFragmentBuilder`) and call the generated class FragmentBuilder so this would be CLEAN.

same to other Builder classes.